spec/services/idv/steps/verify_step_spec.rb

```diff
@@ -65,6 +65,20 @@
       step.call
     end
 
+    context 'when pii_from_doc is not present' do
```

Comment:
Should we have one of these specs for `ssn_step` as well?

Reply:
Yep, added new spec file for it